flang/docs/FlangDriver.md

```diff
@@ -163,6 +163,40 @@ forward compiler options to the frontend driver, `flang-new -fc1`.
 You can read more on the design of `clangDriver` in Clang's [Driver Design &
 Internals](https://clang.llvm.org/docs/DriverInternals.html).
 
+## Linker Driver
```

Comment:
It might be worth adding  an example, e.g.:
```
flang-new file.f90 -###
// editied output here
```

Reply:
Good suggestion!

Reply:
I have tried to capture this now.